app/models/cost_query.rb

```diff
@@ -40,7 +40,7 @@ class CostQuery < Report
             nil
         end.compact
 
-        CostQuery.update_all ["serialized = ?", YAML::dump(serialized)], ["id = ?", query.id]
+        CostQuery.where(["id = ?", query.id]).update_all ["serialized = ?", YAML::dump(serialized)]
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.